ingresslibchart/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
```

Comment:
Was looking over this agin in conjunction with #456 and just wanted to note that these files here might be under the wrong directory, since they aren't under the `charts` directory